packages/cosmic-proto/package.json

```diff
@@ -32,30 +32,10 @@
       "types": "./dist/codegen/cosmos/*.d.ts",
       "default": "./dist/codegen/cosmos/*.js"
     },
-    "./cosmos/bank/v1beta1/query.js": {
-      "types": "./dist/codegen/cosmos/bank/v1beta1/query.d.ts",
-      "default": "./dist/codegen/cosmos/bank/v1beta1/query.js"
-    },
-    "./cosmos/distribution/v1beta1/tx.js": {
-      "types": "./dist/codegen/cosmos/distribution/v1beta1/tx.d.ts",
-      "default": "./dist/codegen/cosmos/distribution/v1beta1/tx.js"
-    },
-    "./cosmos/staking/v1beta1/tx.js": {
-      "types": "./dist/codegen/cosmos/staking/v1beta1/tx.d.ts",
-      "default": "./dist/codegen/cosmos/staking/v1beta1/tx.js"
-    },
-    "./cosmos/tx/v1beta1/tx.js": {
-      "types": "./dist/codegen/cosmos/tx/v1beta1/tx.d.ts",
-      "default": "./dist/codegen/cosmos/tx/v1beta1/tx.js"
-    },
     "./google/*.js": {
       "types": "./dist/codegen/google/*.d.ts",
       "default": "./dist/codegen/google/*.js"
     },
-    "./google/protobuf/any.js": {
-      "types": "./dist/codegen/google/protobuf/any.d.ts",
-      "default": "./dist/codegen/google/protobuf/any.js"
-    },
```

Comment:
Out of scope for this PR?

Reply:
these are all redundant with the subpatterns above them

Reply:
hmm, you're right tho. [breaking](https://github.com/Agoric/agoric-sdk/actions/runs/9038366546/job/24839392140?pr=9356). Oh, I forgot our bundler doesn't support wildcards yet!